ui/components/app/confirm-page-container/confirm-page-container-warning/confirm-page-container-warning.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
```

Comment:
Is this component only going to be used for `setApprovalForAll` confirmations? If so, we should name the component and the file in a way that communicates that (e.g. `set-approval-for-all-warning.js`) and probably move it to `ui/components/app`

Reply:
Done